src/keywords/mod.rs

```diff
@@ -260,4 +260,13 @@ mod tests {
         let compiled = JSONSchema::compile(&schema, None).unwrap();
         assert!(compiled.is_valid(&data))
     }
+
+    #[test_case(json!({"minimum": 18014398509481984u64}), json!(18014398509481984u64 - 1))]
+    #[test_case(json!({"maximum": 18014398509481984u64 - 1}), json!(18014398509481984u64))]
+    fn is_not_valid(schema: Value, instance: Value) {
```

Comment:
I would suggest moving this tests into the correct keyword files.

By having this sort of unit-tests in the specific files allows you to have the test of the corner-cases closer to where you're implementing them


Not blocking ... is more about style :)

Reply:
:+1: Thanks!